File: Day-20/DecimalToBinary.py
import math
n = int(input())
def DecToBin(n):
    answer = 0
    i = 0
    while(n!=0):
        digit = n & 1
        answer = (digit *math.pow(10, i)) + answer
        n = n >>1
        i += 1
    
    print(math.floor(answer))


DecToBin(n)

File: Day-20/DecimalToBinary.js
function DecimalToBinary(n) {
  let answer = 0;
  for (let i = 0; n != 0; i++) {
    let bit = n & 1;
    answer = bit * Math.pow(10, i) + answer;
    n = n >> 1;
  }
  console.log(answer);
}

DecimalToBinary(5);
